app.js

```diff
@@ -36,57 +34,34 @@ startPollButton.addEventListener('click', () => {
     optionB = optionBEl;
     question = questionEl;
     // reset the form values
-    currentQuestionInputEl.value = '';
-    optionAInputEl.value = '';
-    optionBInputEl.value = '';
+    //currentQuestionInputEl.value = '';
+    //optionAInputEl.value = '';
+    //optionBInputEl.value = '';
     // refresh the current game element with new data by calling the appropriate function
     
-
-    console.log('start poll button pressed');
-    const newPoll = {
-        question: question,
-        optionA: optionA,
-        optionB: optionB,
-        votesA: votesA,
-        votesB: votesB,
-    };
-    pastPolls.push(newPoll);
-    pastPolls.textContent = '';
-    for (let poll of pastPolls) {
-        const pollEl = renderPoll(poll.question, poll.optionA, poll.optionB, poll.votesA, poll.votesB);
-        currentPollEl.append(pollEl); 
-
-    }
     refreshCurrentPollEl();
-    question = '';
-    optionA = '';
-    optionB = '';
-    votesA = 0;
-    votesB = 0;
 
-    refreshCurrentPollEl();
-    displayAllPolls();
 });
 
 increaseOptionAButton.addEventListener('click', () => {
     votesA++;
     console.log(votesA);  
```

Comment:
In general it is a good practice to remove any console.logs() from your code before pushing

Reply:
ah missed that, thanks!